src/Common/Compat.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I can do that, although when I tried it caused some problems with apicompat since it couldn't find the assembly anymore, which obviuosly is fixable but I just figured this was easier, but I can remove this and try the other way